src/main/java/com/mongodb/kafka/connect/sink/cdc/debezium/DebeziumCdcHandler.java

```diff
@@ -45,6 +47,11 @@ protected void registerOperations(final Map<OperationType, CdcOperation> operati
 
   public CdcOperation getCdcOperation(final BsonDocument doc) {
     try {
+
+      if (!doc.containsKey(OPERATION_TYPE_FIELD_PATH) && doc.containsKey(DDL_FIELD_PATH)) {
+        return NOOP_CDC_OPERATION;
```

Comment:
Optional: Is it worth adding a test that confirms an `op` will pass through, even when there is a `ddl`?